### PR TITLE
CFN Stack timestamps attributes

### PR DESCRIPTION
**Pull request by pinzon**
This PR  addresses issue #6038. Due to some difficulties with Commandeer while triaging, I couldn't exactly find out why it still displays a stack with the state "DELETE_COMPLETE", my best guess is that it uses the timestamp attributes `["CreationTime", "DeletionTime", "LastUpdatedTime"]` to determine if the stack should be shown but Localstack does not return the "DeletionTime" or the "LastUpdatedTime".

Changes:
- Set the DeletionTime and LastUpdatedTime attributes accordingly.
- Test to validate the return of the attributes when calling the cfn describe stack function.

### CODE REVIEW

File: tests/integration/cloudformation/test_cloudformation_stacks.py [hunk 1]
```diff
@@ -139,3 +139,35 @@ def wait_stack_done():
     assert wait_until(wait_stack_done)
     stack_describe_response = cfn_client.describe_stacks(StackName=stack_id)["Stacks"][0]
     assert len(stack_describe_response["Outputs"]) == 2
+
+
+def test_stack_time_attributes(cfn_client, deploy_cfn_template):
+    bucket_name = "test"
```
Comment (dominikschubert) [inline tests/integration/cloudformation/test_cloudformation_stacks.py:145]:
```suggestion
@pytest.mark.aws_validated
def test_stack_time_attributes(cfn_client, deploy_cfn_template):
    bucket_name = f"test-{short_uid()}"
```

Please test this against AWS and add the annotation when validated successfully :+1: 

The bucket_name not being unique leads to issues when deploying on AWS.

File: tests/integration/cloudformation/test_cloudformation_stacks.py [hunk 2]
```diff
@@ -139,3 +139,35 @@ def wait_stack_done():
...
+        parameters={"LocalParam": bucket_name},
+    )
+    stack_name = deployed.stack_name
+    stack_id = deployed.stack_id
+
+    assert "CreationTime" in cfn_client.describe_stacks(StackName=stack_name)["Stacks"][0]
+
+    cfn_client.update_stack(
+        StackName=stack_name,
+        TemplateBody=load_template_raw("template5.yaml"),
```
[the remark this answers is not in the document]
Comment (dominikschubert) [reply tests/integration/cloudformation/test_cloudformation_stacks.py:157]: load_template_raw is being outphased

**Review by dominikschubert — CHANGES_REQUESTED** [review]
Nice to see more detailed parity for stack operations :+1: 

Encountered some issues with the test that we should solve before merging but otherwise lgtm

**Comment by pinzon** [comment]
Hi @dominikschubert, considering your comments, I added a simple stack that has only an ApiGateway.RestApi  and the update only changes the name property. Also the test should have all the suggested changes.

**Comment by dominikschubert** [comment]
Hi @pinzon ! Please rebase on to latest master then we're good to go :+1: 
Just need to check that we don't break the CI since we had some test fixture refactoring in the meantime!

**Comment by pinzon** [comment]
hi @dominikschubert , I just rebased the branch. :+1:

**Review by dominikschubert — APPROVED** [review]
LGTM! :+1:

**Pull request merged**